flask_ask/models.py

```diff
@@ -106,7 +106,8 @@ def render_response(self):
             kw[kwargname] = json_encoder
         _dbgdump(response_wrapper, **kw)
 
-        return json.dumps(response_wrapper, **kw)
+        return json.dumps(response_wrapper,
+                          indent=2 if current_app.config.get('ASK_MINIFY_DEBUG_LOGS', False) else None, **kw)
```

Comment:
Oh, that's not even supposed to be a debug log, that's an actual Alexa response. Removing the indentation arg entirely.